vcenter_operator/masterpassword.py

```diff
@@ -1,3 +1,4 @@
+import sys
```

Comment:
You don't need that import anymore, do you?

Reply:
sorry